### DISCUSSION TITLE
How do I properly add text attributes to tables in Quarto?

### DISCUSSION THREAD

**Discussion by Jarartur** [quarto-dev/quarto-cli#6132] [Q&A]
### Description

Hi, I could find any examples that work and it is not in the documentation. I do not have any real experience with HTML so pardon me if it's something trivial I do not understand.

Basically I want to create a table that has some cells joined, so I figured I need to use html table. Now I want to center the text in those joined cells. I understand that this is done by the `style="text-align: center;"` attribute. But after rendering the document in pdf it does not work this way.

```{=html}
<table>
<thead>
  <tr>
    <th>Artifact</th>
    <th>No. of train artifacts</th>
    <th>No. of validation artifacts</th>
    <th>Accuracy (%)</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>Low Blur</td>
    <td style="text-align: center;" colspan="2">Generated</td>
    <td>43</td>
  </tr>
  <tr>
    <td>High Blur</td>
    <td style="text-align: center;" colspan="2">Generated</td>
    <td>57</td>
  </tr>
</tbody>
</table>
```

But after rendering it shows as:
![image](https://github.com/quarto-dev/quarto-cli/assets/45856425/4840f5f4-441f-4616-916e-4cb9500e3966)

**Comment by mcanouil** [comment]
You can't use CSS (it's for HTML) for LaTeX/PDF.
Pandoc will completely ignore that.

See what you can do using raw HTML table at <https://quarto.org/docs/authoring/tables.html#html-tables>.
